src/credentials/CHIPOperationalCredentials.cpp

```diff
@@ -364,5 +364,24 @@ P256Keypair * OperationalCredentialSet::GetNodeKeypairAt(const CertificateKeyId
     return nullptr;
 }
 
+const ChipCertificateData * OperationalCredentialSet::GetRootCertificate(const CertificateKeyId & trustedRootId) const
+{
+    for (uint8_t i = 0; i < mOpCredCount; i++)
```

Comment:
Please change any indexing using uint8_t to size_t. This is, as @msandstedt pointed-out, an artificial limitation on maximum number of certs available in these primitive building block classes.

Reply:
The loop indexing can be changed today to size_t for `i` and `j`, even if the field doesn't change, because size_t is a wider type

Reply:
Updated to use size_t and new var names